### PR TITLE
Update the tag patter in the docs

### PR DESCRIPTION
**Pull request by degiz**
**Proposed changes**:

- The docs can now point to the `2.0.0a1` `2.0.0rc1` version

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by tmbo** [comment]
mhm I think we usually use the `2.0.0a1` and `2.0.0rc1` format (as that is the python default)

**Comment by degiz** [comment]
Shouldn't it be the semver style?

**Comment by tmbo** [comment]
yes imo it should be semver, but the python community decided it would be a great idea to go a different route when handling pre-release versions...

File: docs/conf.py [hunk 1]
```diff
@@ -353,6 +353,7 @@
 scv_grm_exclude = ("README.md", ".gitignore", ".nojekyll", "CNAME")
 scv_whitelist_tags = (
     re.compile(r"^[2-9]+\.\d+\.\d+$"),
+    re.compile(r"^2\.[0-9]+\.[0-9]?(\w+)?$"),
```
Comment (tmbo) [inline docs/conf.py:356]:
```suggestion
    re.compile(r"^2\.[0-9]+\.[0-9]+(\w+)?$"),
```

**Pull request merged**